docker.go

```diff
@@ -265,6 +285,64 @@ func (c *DockerContainer) Exec(ctx context.Context, cmd []string) (int, error) {
 	return exitCode, nil
 }
 
+func (d *DockerContainer) startLogProducer() {
+	d.stopProducer = make(chan bool)
+	go func() {
```

Comment:
I am ignoring errors that occur in this goroutine, I believe I would just want to log them out, and continue running the goroutine.  is there a better way to handle errors in this goroutine?